google/cloud/pubsub/options.h

```diff
@@ -220,12 +220,38 @@ struct CompressionAlgorithmOption {
   using Type = int;
 };
 
+/**
+ * The maximum number of Open Telemetry span links.
```

Comment:
I presume this isn't really PubSub specific, and am some point we'll want to move it out.  Should we just do that now?  Or is that what we decided to leave as-is for now in the meeting yesterday?

Reply:
We decided yesterday leave as is Pub/Sub specific for now. If another library wants it, we can use move it.